src/alignments.cpp

```diff
@@ -25,9 +25,9 @@ void paf_worker(
             // Query/Target end (0-based; BED-like; open)
             paf.target_start >= paf.target_sequence_length || paf.target_end > paf.target_sequence_length || paf.target_start >= paf.target_end) break;
         size_t query_idx = seqidx.rank_of_seq_named(paf.query_sequence_name);
-        size_t query_len = seqidx.nth_seq_length(query_idx);
```

Comment:
are these unused?

Reply:
Yes, they weren't used. I left them as comments in case they will be needed in the future.